pages/c/_cluster/apps/install.vue

```diff
@@ -512,6 +526,17 @@ export default {
         this.$fetch();
       }
     },
+
+    'value.metadata.namespace'(neu, old) {
+      if (neu) {
+        const ns = this.$store.getters['cluster/byId'](NAMESPACE, this.value.metadata.namespace);
+        const project = ns.metadata.annotations[PROJECT];
```

Comment:
When viewing `External IP Webhook` or `Logging` (I don't have either installed) I get...
```
TypeError: Cannot read property 'metadata' of undefined
    at VueComponent.valueMetadataNamespace (install.vue?ab9f:535)
```
Switching this line to `const project = ns?.metadata.annotations[PROJECT];` works, but I didn't delve into the existing namespace problem to understand if this is correct fix.

Reply:
This is a good fix. The error happens when the chart specifies a namespace that doesn't exist yet. 
It took me a minute to track this down: I'd thought `value.metadata.namespace` was set in `fetch` when the chatInstallAction is created (this watcher not triggered) but it's actually set by the `forceNamespace` functionality of NameNsDescription after `this.value` store object has been created (so the watcher is triggered).